src/daria/corrections/color/colorcorrection.py

```diff
@@ -115,22 +118,21 @@ def __call__(
         """
         # Make sure that the image is in uint8 or uint16 format
         if image.dtype in [np.uint8, np.uint16]:
-            image = skimage.util.img_as_float(image)
+            image = skimage.img_as_float(image)
```

Comment:
I am assuming the methods are identical, or?

Reply:
I think they are the same (at least everything looks the same), but for some reason python complains that the .util part is old and not supposed to be used anymore..

Reply:
Oh interesting! Good to know.